Resources/views/BreadCrumb/breadCrumb.html.twig

```diff
@@ -3,6 +3,9 @@
 {% block list %}
   {% if item.hasChildren and options.depth is not same as(0) and item.displayChildren %}
       <ol class="breadcrumb">
+        <li data-button="back" class="breadcrumb-item back">
```

Comment:
Als je hier kan op klikken zou ik dat wel op een anchor of button zetten. Ben ook niet helemaal overtuigd over de benaming van het data attribuut. `data-button-previous` is misschien beter, al ben ik ook niet helemaal overtuigd

Reply:
Ik zou ook gaan voor `breadcrumb-item-back` ipv `back`